presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestDistributedQueries.java

```diff
@@ -377,6 +380,8 @@ public void testDropColumn()
     @Test
     public void testAddColumn()
     {
+        featuresToTest(TestedFeature.ADD_COLUMN);
```

Comment:
i think this should be named declaratively `testedFeatures` (or `requiredFeatures`, see comment in cassandra commit) 

 Also, this could be annotation-based (`@TestedFeatures(ADD_COLUMN)`), I think this is super-easy to add annotation-based exclusion and can help.

Reply:
`@TestedFeatures(ADD_COLUMN)` lets do not over complicate things from start. First let see how much this framework is useful.

Renamed to `testedFeatures`.